cypress/integration/rendering/flowchart-v2.spec.js

```diff
@@ -610,4 +610,27 @@ flowchart RL
       {htmlLabels: true, flowchart: {htmlLabels: true}, securityLevel: 'loose'}
```

Comment:
@knsv It looks like the `Resolves xxx` statement in the issue description only closed the first issue in the list when this merged.

Thanks for great addition! ❤️